Automattic/wp-calypso
# Design Picker: Switch to `react-intersection-observer`

See https://github.com/Automattic/wp-calypso/issues/74022
Previously https://github.com/Automattic/wp-calypso/pull/71064

## Proposed Changes

Switches to `react-intersection-observer` in the Design Picker because `calypso/lib/use-in-view` is buggy and it doesn't make sense to spend time reinventing the wheel.

## Testing Instructions

1. Create a new Premium site.
2. Hit `Continue` until you get to the Design Picker.
3. Click on a category and verify `calypso_signup_design_scrolled_to_end` is only fired once when you hit the end of the page (even if you scroll to the bottom of the page multiple times).

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/36432/231313096-fd61cea6-dcff-4121-8683-1120d960e91a.png">
